stub-environment/cluster.txt

```diff
@@ -1,3 +1,4 @@
+test-laptop 08:00:27:56:A2:21 10.0.100.5 - bcpc_host bcpc.example.com SKIP
```

Comment:
This entry adds a `VIP` host to `cluster.txt` and since it has `SKIP` in its role section no `cluster-assing-role` operation will be performed. This is needed to generate `keytabs` for vip host when cluster is being built.